<!DOCTYPE html>
<html lang="en">

<head>

    <title>Yuanning-Li Lab</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!--FABICON ICON -->
    <link rel="icon" href="assets/img/icons/li_logo_big.png">

    <!-- ALL CSS FILES -->

    <link rel="stylesheet" href="assets_2/css/main.css" />

    <link rel="stylesheet" href="assets/css/plugins/bootstrap.min.css">
    <link rel="stylesheet" href="assets/css/plugins/fontawesome.css">
    <link rel="stylesheet" href="assets/css/plugins/owl.carousel.min.css">
    <link rel="stylesheet" href="assets/css/plugins/modal-video.min.css">
    <link rel="stylesheet" href="assets/css/plugins/stellarnav.css">
    <link rel="stylesheet" href="assets/css/custom_animation.css">
    <link rel="stylesheet" href="assets/css/main.css">
    <link rel="stylesheet" href="assets/css/typography.css">
    <link rel="stylesheet" href="assets/css/btn.css">
    <link rel="stylesheet" href="assets/css/inner_pages.css">
    <link rel="stylesheet" href="assets/css/theme.css">
    <link rel="stylesheet" href="assets/css/responsive.css">
</head>

<body>
    <!-- Preloader -->
    <div class="preloader">
        <div class="spinner_wrap">
            <div class="spinner"></div>
        </div>
    </div>
    <!--preload area end-->

    <!---====sidebar area start====-->
    <div class="sidebar__area">
        <div class="sidebar__single">
            <div class="clase__btn"> <i class="far fa-times"></i>
            </div>
            <div class="sidebar__content">
                <div class="sidebar__logo__area">
                    <a href="#" class="siebar__logo">
                        <img src="assets/img/icons/li_logo.png" alt="">
                    </a>
                </div>
                <div class="sidebar__text">
                    <p>Diversity of eukaryotics is far greater than most people realize. In the Yuanning-Li lab, we are trying to use DNA records to study patterns and process of eukaryotic evolution on different time scales and with different types of genomic information. Combining a molecular systematic and/or genomic approach with information from organismal evolution has proved a powerful approach to study everything from the origin of major eukaryotic lineages (especially for marine invertebrates and fungi) to the recent adaptation to deep-sea chemosynthetic environments. Research in the lab focuses on utilizing computational approaches and genome-scale data to gain insight into molecular phylogenetics, evolutionary biology, comparative genomics, and bioinformatics.

                    </p>
                </div>
                <div class="sidebar__socials">
                </div>
            </div>
        </div>
    </div>
    <!---====sidebar area end====-->

        <!---====menu area start====-->
        <div class="menu-area" id="myHeader">
            <!---====srollup btn====--> <a href="#" class="up_btn"><i class="far fa-long-arrow-alt-up"></i></a>
            <div class="container">
                <div class="row">
                    <div class="col-6 col-lg align-self-center">
                        <div class="logo">
                            <a href="index.html">
                                <img src="assets/img/icons/li_logo_big.png" alt="logo">
                            </a>
                        </div>
                    </div>
                    <div class="col-lg-2Â· text-right align-self-center d-none d-lg-block">
                        <div class="case_btn_select">
                            <ul>
                                <li><a href="#">What We Do<i class="fal fa-angle-down"></i></a>
                                    <ul>
                                        <li><a href="whatdo2.html">Eukaryotic Groups</a>
                                        </li>
                                        <li><a href="whatdo2.html">Deep-sea Chemosynthetic Holobionts</a>
                                        </li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                    </div>
                    <div class="col-5 col-lg-5 text-center align-self-center">
                        <div class="main-menu d-inline stellarnav">
                            <ul class="navbar">
                                <li><a href="index.html">Home </a>
                                </li>
                                <li><a href="index.html">About</a>
                                </li>
                                <li><a href="index.html">News</a>
                                </li>
                                <li><a href="#">Lab <i class="fal fa-plus"></i></a>
                                    <ul>
                                        <li><a href="about.html">People</a></li>
                                        <li class="current"><a href="publications.html">Publications</a></li>
                                        <li><a href="life.html">Life & Story</a></li>
                                    </ul>
                                </li>
                                <li><a href="contact.html">Contact</a>
                                </li>
                            </ul>
                        </div>
                    </div>
                    <div class="col-lg align-self-center d-none d-lg-block text-right">
                        <div class="header_submenu">
                            <ul>
                                <li class="sidebar__open">
                                    <button type="button"><i class="fal fa-bars"></i>
                                    </button>
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            </div>
        </div>

        <!---====menu area end====-->

    <!---====welcome area start====-->
    <div class="welcome-area inner inner black_bg bg inner_bg">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <div class="welcome_text">
                        <h1>Publications</h1>
                        <a class="back_top_home" href="index.html">Home</a> &nbsp; | &nbsp; Publications</div>
                </div>
            </div>
        </div>
    </div>
    <!---====welcome area end====-->

    <!---====about area start====-->
    <div class="about-area inner section-padding">
        <div class="container">

            <div class="row">
                <div>
                    <h3 class="alignleft">Citations are available at Google Scholar.<br>Lab members are indicated in <b><font face="verdana">bold</font></b>. # co-first author, * corresponding author.</h2>
                </div>
            </div>

            <div class="space-100"></div>

            <div class="row">
                <div>
                    <h3 class="alignleft color_orange">Preprints/Submitted articles</h2>
                </div>
            </div>

            <div class="space-20"></div>
            <div class="border-_2px_black"></div>
            <div class="space-40"></div>
                <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                    2.<b><font face="verdana">Feng, B.</font></b>#, Yo. Li#, <b><font face="verdana">H. Liu</font></b>#, J. L. Steenwyk, K. T. David, <b><font face="verdana">X. Tian</font></b>, <b><font face="verdana">B. Xu</font></b>, C. Goncalves, D. A. Opulente, A. L. LaBella, M.-C. Harrison, J. F. Wolters, <b><font face="verdana">S. Shao</font></b>, <b><font face="verdana">Z. Chen</font></b>, K. J. Fisher, M. Groenewald, C. T. Hittinger, X.-X. Shen, A. Rokas*, X. Zhou*, <b><font face="verdana">Y. Li*</font></b>. Unique trajectory of gene family evolution from genomic analysis of nearly all known species in an ancient yeast lineage.
                    <br><a class="alignright" href="https://www.biorxiv.org/content/10.1101/2024.06.05.597512v1">Link </a><br>
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.06.05.597512" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1101/2024.06.05.597512" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

                    <div class="space-80"></div>

                    <div class="space-40"></div>
                        <div class="author_text2">
                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                            1.<b><font face="verdana">Liu, H.</font></b>#,  Steenwyk JL#, Schultz D, Kocot KM,  Zhou X, Shen XX, Zhao T, Rokas A.*, <b><font face="verdana">Li Y</font></b>*. A genome-scale Opisthokonta tree of life: toward phylogenomic resolution of ancient divergences.
                        </h4>
                            <div class="alignright" style="padding-left: 1em;">
                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                            <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.09.20.556338" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                            <div data-badge-type="1" data-doi="10.1101/2023.09.20.556338" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                            </div>

                    
            <div class="author_text2">
                <h4  style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                    
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="#" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="#" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>
                </h4>
            </div>

            <div class="space-40"></div><div class="space-40"></div>


            <div class="row">
                <div>
                    <h3 class="alignleft color_orange">Research articles</h2>
                </div>
            </div>

            <div class="space-20"></div>
            <div class="border-_2px_black"></div>
            <div class="space-40"></div>

            <div class="row">
                <div>
                    <h5 class="alignleft color_orange"><b><font face="verdana">2024</font></h5>
                </div>
            </div>

            <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                    33.Opulente#, D. A., A. L. LaBella#, M.-C. Harrison, J. F. Wolters, C. Liu, Y. Li, J. Kominek, J. L. Steenwyk, H. R. Stoneman, J. VanDenAvond, C. R. Miller, Q. K. Langdon, M. Silva, C. Goncalves, E. J. Ubbelohde, <b><font face="verdana">Y. Li</font></b>, K. V. Buh, M. Jarzyna, M. A. B. Haase, C. A. Rosa, N. Cadez, D. Libkind, J. H. DeVirgilio, A. B. Hulfachor, C. P. Kurtzman, J. P. Sampaio, P. Goncalves, X. Zhou, X.-X. Shen, M. Groenewald, A. Rokas*, & C. T. Hittinger*. 2024.
                    Genomic factors shape carbon and nitrogen metabolic niche breadth across Saccharomycotina yeasts. <b><font face="verdana">Science.</font></b> <br><a class="alignright" href="pdfs/Opulente et al. 2024 - Genomic factors shape carbon and nitrogen metabolic niche breadth across Saccharomycotina yeasts.pdf">PDF </a><br>
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi=" 10.1126/science.adj4503" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi=" 10.1126/science.adj4503" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

                <div class="space-100"></div>

            <div class="row">
                <div>
                    <h5 class="alignleft color_orange"><b><font face="verdana">2023</font></h5>
                </div>
            </div>

            <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                    32.Steenwyk JL, <b><font face="verdana">Y. Li</font></b>, X. Zhou, XX Shen, Rokas A.*2023.
                    Incongruence in the phylogenomics era. <b><font face="verdana">Nature Reviews Genetics.</font></b> 27:1-7<br><a class="alignright" href="pdfs/JL_2023_natreviw.pdf">PDF </a><br>                 
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi=" https://doi.org/10.1038/s41576-023-00620-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi=" https://doi.org/10.1038/s41576-023-00620-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

            <div class="space-100"></div>

            <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                    31.M. Groenewald*, C.T. Hittinger, K. Bensch, D.A. Opulente, X.-X. Shen, <b><font face="verdana">Li Y.</font></b>, C. Liu, A.L. LaBella, X. Zhou, S. Limtong, S. Jindamorakot,
                    P. GonÃ§alves, V. Robert, K.H. Wolfe, C.A. Rosa, T. Boekhout, N. ÄadeÅ¾, G. PÃ©ter, J.P. Sampaio, M.-A. Lachance, A.M. Yurkov, H.-M. Daniel, M. Takashima, K. Boundy-Mills, D. Libkind, K. Aoki, T. Sugita, A. Rokas.
                    2023. A genome-informed higher rank classification of the biotechnologically important fungal subphylum Saccharomycotina. <b><font face="verdana">Studies in Mycology.</font></b> <br><a class="alignright" href="pdfs/Li_2023_taxonomy.pdf">PDF </a><br>                 
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi=" https://doi.org/10.3114/sim.2023.105.01" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi=" https://doi.org/10.3114/sim.2023.105.01" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

            <div class="space-100"></div>
            <div class="row">
                <div>
                    <h5 class="alignleft color_orange"><b><font face="verdana">2022</font></h5>
                </div>
            </div>
            

                <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px ">
                    30.<b><font face="verdana">Li Y.</font></b>,* <b><font face="verdana">Liu H.</font></b>, Steenwyk JL, LaBella AL, Harrison MC, Groenewald M, Zhou X, Shen XX, Zhao T, Hittinger CT, Rokas A.* 2022. Contrasting modes of macro and microsynteny evolution in a eukaryotic subphylum. <b><font face="verdana">Current Biology</font></b> 29:S0960-9822(22)01670-0.<br><a class="alignright" href="pdfs/1-s2.0-S0960982222016700-main.pdf">PDF </a><br>                 
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2022.10.025" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1016/j.cub.2022.10.025" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

            <div class="space-100"></div>

                <div class="author_text2">
                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                    29.Steenwyk JL,* Goltz DC, Buida TJ III, <b><font face="verdana">Li Y.</font></b>, Shen XX, Antonis Rokas.* 2022. OrthoSNAP: A tree splitting and pruning algorithm for retrieving single-copy orthologs from gene family trees. <b><font face="verdana">PLOS Biology</font></b> 20(10): e3001827. <br><a class="alignright" href="./pdfs/journal.pbio.3001827.pdf">PDF </a><br/>
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001827" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001827" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

                <div class="space-100"></div>

                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                    28.<b><font face="verdana">Li Y.</font></b>, Altamia MA., Shipway JR., Brugler MR., Bernardino AF., de Brito TL., Lin Z., da Silva Oliveira FA., Sumida P., Smith CR., Trindade-Silva A, Halanych KM.*, Distel DL.* 2022. Contrasting Modes of Mitochondrial Genome Evolution in Sister Taxa of Wood-Eating Marine Bivalves (Teredinidae and Xylophagaidae). <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 14(6):evac089. <br><a class="alignright" target="_blank" href="pdfs/2022_Li_etal_GBE.pdf">PDF</a><br/>
                    <div  class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evac089" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1093/gbe/evac089" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>
                </h4>

                <div class="space-70"></div>

                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                    27.He C., Liu J.*, Wang R., <b><font face="verdana">Li Y.</font></b>, Zheng Q., Jiao F., He C., Shi Q., Xu Y., Zhang R., Thomas H., et al. 2022. Metagenomic evidence for the microbial transformation of carboxyl-rich alicyclic molecules: A long-term macrocosm experiment. <b><font face="verdana"><i>Water Research</i></font></b> 216, 118281. <br><a class="alignright" target="_blank" href="pdfs/2022_He_etal_Water_Research.pdf">PDF</a><br/>
                </h4>
                    <div class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.watres.2022.118281" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1016/j.watres.2022.118281" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>
                </h4>

                <div class="space-100"></div>

                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                    26.Steenwyk, J. L.*, T. J. Buida III, C. Goncalves, D. C. Goltz, G. H. Morales, M. E. Mead, A. L. LaBella, C. M. Chavez, J. E. Schmitz, M. Hadjifrangiskou, <b><font face="verdana">Y. Li</font></b>, & A. Rokas*. 2022. BioKIT: a versatile toolkit for processing and analyzing diverse types of sequence data. <b><font face="verdana"><i>Genetics</i></font></b> 221(3):iyac079. <br><a class="alignright" target="_blank" href="pdfs/2022_Steenwyk_etal_Genetics.pdf">PDF</a><br/>
                </h4>
                    <div  class="alignright" style="padding-left: 1em;">
                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                    <span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyac079" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                    <div data-badge-type="1" data-doi="10.1093/genetics/iyac079" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                    </div>

                    <div class="space-100"></div>

                    <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                        25.Liu D., Chen J., Wang Y., Meng Y., <b><font face="verdana">Li Y.</font></b>, Huang R., Xia Y., Liu H., Jiao N., Xun L., Liu J.* 2022. <i>Synechococcus sp. PCC7002</i> Uses Peroxiredoxin to Cope with Reactive Sulfur Species Stress. <b><font face="verdana"><i>Mbio</i></font></b> e01039-22.  <br><a class="alignright" target="_blank" href="pdfs/2022_Liu_etal_mBio.pdf">PDF</a><br/>
                    </h4>
                        <div  class="alignright" style="padding-left: 1em;">
                        <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                        <span class="__dimensions_badge_embed__" data-doi="10.1128/mbio.01039-22" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                        <div data-badge-type="1" data-doi="10.1128/mbio.01039-22" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                        </div>

                    <div class="space-100"></div>


                    <div class="row">
                        <div>
                            <h5 class="alignleft color_orange"><b><font face="verdana">2021</b></font></h5>
                        </div>
                    </div>

                    <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                        24.<b><font face="verdana">Li Y.</font></b>, Shen X.-X., Evans B., Dunn C.W.*, Rokas A.* 2021. Rooting the animal tree of life. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 38: 4322â4333.  <br><a class="alignright" target="_blank" href="pdfs/2021_Li_etal_MBE.pdf">PDF</a><br/>
                    </h4>
                        <div  class="alignright" style="padding-left: 1em;">
                        <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                        <span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msab170" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                        <div data-badge-type="1" data-doi="10.1093/molbev/msab170" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                        </div>

                        <div class="space-100"></div>


                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                            23.<b><font face="verdana">Li Y.</font></b>, Steenwyk J., Chang Y., Wang Y., James T.Y., Stajich J.E., Spatafora J.W., Groenewald M., Dunn C., Hittinger C.T., Shen X.-X.*, Rokas A.* 2021. A genome-scale phylogeny of the kingdom Fungi. <b><font face="verdana"><i>Current Biology</i></font></b> 31: 1653-1665.  <br> <a class="alignright" target="_blank" href="pdfs/2021_Li_etal_CB.pdf">, PDF</a> <a> </a> <a class="alignright" target="_blank" href="https://figshare.com/articles/dataset/Scripts_and_analyses_used_for_the_fungal_phylogeny/12751736">, Download phylogenomic data</a> <a class="alignright" target="_blank" href="https://doi.org/10.5281/zenodo.3970286">genome data</a><br/>
                            <a class="alignright" target="_blank" href="pics/li_CB_cover.tif.jpg"><b>Cover article</b></a>
                        </h4>
                        <br>
                        <div  class="alignright" style="padding-left: 1em;">
                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                            <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2021.01.074" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                            <div data-badge-type="1" data-doi="10.1016/j.cub.2021.01.074" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                        </div>

                        <div class="space-100"></div>

                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                            22. Novitsky V., Steingrimsson J., Howison M., Dunn C., Gillani, F.S. Manne, A., <b><font face="verdana">Li Y.</font></b>, Spence M. Parillo, Z., Fulton J. and Marak T., 2021. Longitudinal typing of molecular HIV clusters in a statewide epidemic. <b><font face="verdana"><i>AIDS</i></font></b> 35(11):1711-1722.<br> <a class="alignright" target="_blank" href="pdfs/Longitudinal_typing_of_molecular_HIV_clusters_in_a.1.pdf">PDF</a>
                        </h4>
                        <br>
                        <div  class="alignright" style="padding-left: 1em;">
                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                            <span class="__dimensions_badge_embed__" data-doi="10.1097/QAD.0000000000002953" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                            <div data-badge-type="1" data-doi="10.1097/QAD.0000000000002953" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                        </div>

                        <div class="space-100"></div>

                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                            21. Steenwyk, J. L., M. E. Mead, P. A. Castro, C. Valero, A. Damasio, R. A. C. Santos, A. L. LaBella, <b><font face="verdana">Y. Li</font></b>, S. L. Knowles, H. A. Raja, N. H. Oberlies, X. Zhou, O. A. Cornely, F. Fuchs, P. Koehler*, G. H. Goldman*, & A. Rokas*, 2021. Genomic and phenotypic analysis of COVID-19-associated pulmonary aspergillosis isolates of <i>Aspergillus fumigatus</i>. <b><font face="verdana"><i>Microbiology Spectrum</i></font></b> 9(1): e00010-21. <br><a class="alignright" target="_blank" href="pdf_new/2021_Steenwyk_etal_MicrobiolSpectrum.pdf">PDF</a><br/>
                        </h4>
                            <div  class="alignright" style="padding-left: 1em;">
                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                            <span class="__dimensions_badge_embed__" data-doi="10.1128/Spectrum.00010-21 " data-hide-zero-citations="true" data-style="large_rectangle"></span>
                            <div data-badge-type="1" data-doi="10.1128/Spectrum.00010-21 " data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                            </div>

                            <div class="space-100"></div>

                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                20. Steenwyk J., Buida III T.J., LaBella A.L., <b><font face="verdana"> Li Y.</font></b>, Shen X.-X., Rokas A. 2021. PhyKIT: a UNIX shell toolkit for processing and analyzing phylogenomic data. <b><font face="verdana"><i>Bioinformatics</i></font></b> 37(16):2325-2331. <br><a class="alignright" target="_blank" href="pdfs/2021_Steenwyk_etal_PhyKIT_Bioinf.pdf">PDF</a> <br><a class="alignright" target="_blank" href="https://jlsteenwyk.com/h4hyKIT/">Download PhyKIT</a><br/>
                            </h4>
                                <div  class="alignright" style="padding-left: 1em;">
                                <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btab096" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                <div data-badge-type="1" data-doi="10.1093/bioinformatics/btab096" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                </div>

                                <div class="space-100"></div>

                                <div class="row">
                                    <div>
                                        <h5 class="alignleft color_orange"><b><font face="verdana">2020</b></font></h5>
                                    </div>
                                </div>

                                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                    19. Shen X.-X.*, <b><font face="verdana">Li Y.</font></b>,  Hittinger C., Chen X., Rokas A.* 2020. An investigation of irreproducibility in maximum likelihood phylogenetic inference. <b><font face="verdana"><i>Nature Communications</i></font></b> 11:6096. <br><a class="alignright" target="_blank" href="pdfs/2020_Shen_etal_Nat_Comm.pdf">PDF</a> <br><a class="alignright" target="_blank" href="https://figshare.com/articles/dataset/Irreproducibility_of_maximum_likelihood_phylogenetic_inference/11917770">Download data</a><br/>
                                </h4>
                                    <a class="alignright" target="_blank" href="https://natureecoevocommunity.nature.com/h4osts/irreproducibility-in-phylogenetic-inference">The story behind the paper in Nature Ecology & Evolution blog</a>
                                    <br><div  class="alignright" style="padding-left: 1em;">
                                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                    <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-20005-6" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                    <div data-badge-type="1" data-doi="10.1038/s41467-020-20005-6" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                    </div>

                                    <div class="space-100"></div>

                                    <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                        18. Steenwyk J., Buida III T.J., <b><font face="verdana">Li Y.</font></b>, Shen X.-X., Rokas A. 2020. ClipKIT: a multiple sequence alignment-trimming algorithm for accurate phylogenomic inference. <b><font face="verdana"><i>PLoS Biology</i></font></b> 18: e3001007.  <br><a class="alignright" target="_blank" href="pdfs/2020_Steenwyk_etal_ClipKIT_PLoSBiol.pdf">PDF</a>  <br><a class="alignright" target="_blank" href="https://jlsteenwyk.com/ClipKIT/">Download ClipKT</a><br/>
                                    </h4>
                                        <div  class="alignright" style="padding-left: 1em;">
                                        <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                        <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001007" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                        <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001007" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                        </div>

                                        <div class="space-100"></div>

                                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                            17. <b><font face="verdana">Li Y.</font></b>#, David K.T.#, Shen X.-X., Steenwyk J., Halanych K.M., Rokas A. 2020. Feature Frequency Profile-based phylogenies are inaccurate. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> in press. <br><a class="alignright" target="_blank" href="pdfs/2020_Li_etal_PNAS.pdf">PDF</a><br/>
                                        </h4>
                                            <div  class="alignright" style="padding-left: 1em;">
                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                            <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2013143117" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                            <div data-badge-type="1" data-doi="10.1073/pnas.2013143117" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                            </div>

                                            <div class="space-100"></div>

                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                16. Shen X.-X.*, Steenwyk J., LaBella A., Opulente D., Zhou X., Kominek J., <b><font face="verdana">Li Y.</font></b>, Groenewald M., Hittinger C., Rokas A.* 2020. Genome-scale phylogeny and contrasting modes of genome evolution in the fungal phylum Ascomycota. <b><font face="verdana"><i>Science Advances </i></font></b> 6(45):eabd0079. <br><a class="alignright" target="_blank" href="pdfs/2020_Shen_etal_Sci_adv.pdf">PDF</a> <br><a class="alignright" target="_blank" href="https://figshare.com/articles/dataset/h4hylogenomics_and_contrasting_modes_of_genome_evolution_in_Ascomycota/12196149">Download phylogenomic data</a> <br><a class="alignright" target="_blank" href="https://doi.org/10.5281/zenodo.3783970">genome assemblies and annotations</a><br/>
                                            </h4>
                                                <div  class="alignright" style="padding-left: 1em;">
                                                <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                <span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abd0079" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                <div data-badge-type="1" data-doi="10.1126/sciadv.abd0079" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                </div>

                                                <div class="space-100"></div>

                                                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                    15. Novitsky V., Steingrimsson J.A., Howison M., Gillani F.S., <b><font face="verdana">Li Y.</font></b>, Manne A., Fulton J., Spence M., Parillo Z., Marak T. and Chan P.A., 2020. Empirical comparison of analytical approaches for identifying molecular HIV-1 clusters. <b><font face="verdana"><i>Scientific Reports</i></font></b> 10(1):1-11. <br><a class="alignright" target="_blank" href="pdfs/2020_Shen_etal_Sci_adv.pdf">PDF</a><br>
                                                </h4>
                                                    <div  class="alignright" style="padding-left: 1em;">
                                                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                    <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-020-75560-1" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                    </div>
                                                    <br>
                                                <div class="alignright">
                                                    <div data-badge-type="1" data-doi="0.1038/s41598-020-75560-1" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                </div>

                                                <div class="space-100"></div>

                                                <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                    14. Kantor R., Fulton  J.P., Steingrimsson J., Novitsky V., Howison M., Gillani F., <b><font face="verdana">Li Y.</font></b>, Manne A., Parillo Z., Spence M. and Marak T., 2020. Challenges in evaluating the use of viral sequence data to identify HIV transmission networks for public health. <b><font face="verdana"><i>Statistical Communications in Infectious Diseases</i></font></b> 12(S1). <br><a class="alignright" target="_blank" href="pdf_new/2020_kantor_etal.pdf">PDF</a>
                                                </h4>
                                                <br><div  class="alignright" style="padding-left: 1em;">
                                                    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                    <span class="__dimensions_badge_embed__" data-doi="10.1515/scid-2019-0019" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                    <div data-badge-type="1" data-doi="10.1515/scid-2019-0019" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                    </div>

                                                    <div class="space-100"></div>

                                                    <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                        13. Kong L.#, <b><font face="verdana">Li Y.#</font></b>, Kocot K.M., Yang Y., Qi L., Li Q. and Halanych K.M.*, 2020. Mitogenomics reveals phylogenetic relationships of Arcoida (Mollusca, Bivalvia) and multiple independent expansions and contractions in mitochondrial genome size. <b><font face="verdana"><i>Molecular Phylogenetics and Evolution</i></font></b> 150, p.106857. <br><a class="alignright" target="_blank" href="pdf_new/2020_Kong_etal_MPE.pdf">PDF</a><br/>
                                                    </h4>
                                                        <div  class="alignright" style="padding-left: 1em;">
                                                        <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                        <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2020.106857" data-hide-zero-citations="true" data-style="large_rectangle"></span>	
                                                        <div data-badge-type="1" data-doi="10.1016/j.ympev.2020.106857" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                        </div>

                                                        <div class="space-100"></div>

                                                        <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                            12. Zhou Y.*, <b><font face="verdana">Li Y.</font></b>, Cheng H., Halanych K.M. and Wang C.*, 2020. The mitochondrial genome of the bone-eating worm Osedax rubiplumus (Annelida, Siboglinidae). <b><font face="verdana"><i>Mitochondrial DNA Part B</i></font></b> 5(3):2267-2268. <br><a class="alignright" target="_blank" href="pdf_new/2020_Zhou_etal_MitochondrialpartB.pdf">PDF</a><br/>
                                                        </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1080/23802359.2020.1772680" data-hide-zero-citations="true" data-style="large_rectangle"></span>	
                                                            <div data-badge-type="1" data-doi="10.1080/23802359.2020.1772680" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                11. Zhou Y., Wang, <b><font face="verdana">Y. Li</font></b>, Y., Shen C., Liu Z. and Wang C., 2020. First report of Osedax in the Indian Ocean indicative of trans-oceanic dispersal through the Southern Ocean. <b><font face="verdana"><i>Marine Biodiversity</i></font></b> 50(1):1-9. <br><a class="alignright" target="_blank" href="pdf_new/2020_Zhou_etal_MarineBiodiversity.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1007/s12526-019-01034-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>	
                                                            <div data-badge-type="1" data-doi="10.1007/s12526-019-01034-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                10. <b><font face="verdana">Li Y.</font></b>, Tassia M.G., Waits D.S., Bogantes V.E., David K.T. and Halanych K.M., 2019. Genomic adaptations to chemosymbiosis in the deep-sea seep-dwelling tubeworm Lamellibrachia luymesi. <b><font face="verdana"><i>BMC Biology</i></font></b> 17(1):1-14. <br><a class="alignright" target="_blank" href="pdf_new/2018_Li_etal_BMCBiology.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1186/s12915-019-0713-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1186/s12915-019-0713-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4>
                                                                9. Voight J.R., Marshall B.A., Judge J., Halanych K.M., <b><font face="verdana">Li Y.</font></b>, Bernardino A.F., Grewe F. and Maddox J.D., 2019. Life in wood: preliminary phylogeny of deep-sea wood-boring bivalves (Xylophagaidae), with descriptions of three new genera and one new species. <b><font face="verdana"><i>Journal of Molluscan Studies</i></font></b> 85(2):1-9. <br><a class="alignright" target="_blank" href="pdf_new/2019_Voight_etal_MolluscanStudy.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1093/mollus/eyz003" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1093/mollus/eyz003" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                8. <b><font face="verdana">Li Y.</font></b>*, Kocot K.M., Tassia M.G., Cannon J.T., Bernt M. and Halanych K.M.*, 2019. Mitogenomics reveals a novel genetic code in Hemichordata. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 11(1):29-409. <br><a class="alignright" target="_blank" href="pdfs/2020_Li_etal_PNAS.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evy254" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1093/gbe/evy254" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                7. Galaska M.P., <b><font face="verdana">Li Y.</font></b>, Kocot K.M., Mahon A.R. and Halanych K.M.*, 2019. Conservation of mitochondrial genome arrangements in brittle stars (Echinodermata, Ophiuroidea). <b><font face="verdana"><i>Molecular Phylogenetics and Evoluion</i></font></b> 30, pp.115-120. <br><a class="alignright" target="_blank" href="pdf_new/2019_Galaska_etal_MPE.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2018.10.002" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1016/j.ympev.2018.10.002" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                6. Li N., Bao L., Zhou T., Yuan Z., Liu S., Dunham R., <b><font face="verdana">Li Y.</font></b>, Wang K., Xu X., Jin Y. and Zeng Q., 2018. Genome sequence of walking catfish (Clarias batrachus) provides insights into terrestrial adaptation. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 19(1):1-16. <br><a class="alignright" target="_blank" href="pdf_new/2018_Li_etal_BMCBiology.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-018-5355-9" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1186/s12864-018-5355-9" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                5. <b><font face="verdana">Li Y.*</font></b>, Liles M.R. and Halanych K.M.*, 2018. Endosymbiont genomes yield clues of tubeworm success. <b><font face="verdana"><i>The ISME journal</i></font></b> 12(11):2785-2795. <br><a class="alignright" target="_blank" href="pdf_new/2018_Li_etal_ISME.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41396-018-0220-z" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1038/s41396-018-0220-z" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                4. Bernardino A.F., <b><font face="verdana">Li, Y.</font></b>, Smith C.R. and Halanych K.M., 2017. Multiple introns in a deep-sea Annelid (Decemunciger: Ampharetidae) mitochondrial genome. <b><font face="verdana"><i>Scientific Reports</i></font></b> 7(1):1-8. <br><a class="alignright" target="_blank" href="pdf_new/2018_Bernadino_ScientificReports.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-017-04094-w" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1038/s41598-017-04094-w" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                3. <b><font face="verdana">Li Y.*</font></b>, Kocot K.M., Whelan N.V., Santos S.R., Waits D.S., Thornhill D.J. and Halanych K.M.*, 2017. Phylogenomics of tubeworms (Siboglinidae, Annelida) and comparative performance of different reconstruction methods. <b><font face="verdana"><i>Zoologica Scripta</i></font></b> 46(2):200-213. <br><a class="alignright" target="_blank" href="pdf_new/Li_et_al_2016_Siboglinidae_phylogenomics.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                            <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                            <span class="__dimensions_badge_embed__" data-doi="10.1111/zsc.12201" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                            <div data-badge-type="1" data-doi="10.1111/zsc.12201" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                2. Waits D.S., Santos S.R., Thornhill D.J., <b><font face="verdana">Li Y</font></b>. and Halanych K.M.*, 2016. Evolution of sulfur binding by hemoglobin in Siboglinidae (Annelida) with special reference to bone-eating worms, Osedax. <b><font face="verdana"><i> Journal of molecular evolution</i></font></b> 82(4-5):219-229. <br><a class="alignright" target="_blank" href="pdf_new/2016_Waits_etal_JournalOfMolecularEvolution.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                                <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                                <span class="__dimensions_badge_embed__" data-doi="10.1007/s00239-016-9739-7" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                                <div data-badge-type="1" data-doi="10.1007/s00239-016-9739-7" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                            </div>

                                                            <div class="space-100"></div>

                                                            <h4 style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                                                                1. <b><font face="verdana">Li Y.</font></b>, Kocot K.M., Schander C., Santos S.R., Thornhill D.J. and Halanych K.M., 2015. Mitogenomics reveals phylogeny and repeated motifs in control regions of the deep-sea family Siboglinidae (Annelida). <b><font face="verdana"><i> Molecular Phylogenetics and Evolution</i></font></b> 85:221-229. <br><a class="alignright" target="_blank" href="pdf_new/2015_Li_etal_MPE.pdf">PDF</a><br/>
                                                            </h4>
                                                            <div  class="alignright" style="padding-left: 1em;">
                                                                <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
                                                                <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2015.02.008" data-hide-zero-citations="true" data-style="large_rectangle"></span>
                                                                <div data-badge-type="1" data-doi="10.1016/j.ympev.2015.02.008" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
                                                                </div>

                                                                <div class="space-100"></div>

                </div>
            </div>


        </div>
    </div>
    <!---====about area end====-->


    <!---====footer-area area start====-->
    <div class="footer-area footer1 v1 section-padding bg white footer_bg footer_bg1 wbg1 bgaf">
        <div class="container">
            <div class="row">
                <div class="col-12 col-md-4 col-lg-4">
                    <div class="single_footer">
                        <ul>
                            <li><a>Shandong University</a>
                            </li>
                            <li><a>Institute of Marine Science and Technology</a>
                            </li>
                            <br>
                            <li><a>Building K3, 72 Binhai Rd, Qingdao, China</a>
                            </li>
                        </ul>
                    </div>
                </div>
                <div class="col-12 col-md-4 col-lg">
                
                </div>
                <div class="col-12 col-md-4 col-lg">
                
                </div>
                <div class="col-12 col-md-4 col-lg-6">
                    <img src="assets/img/foot.png" alt="">
                </div>
            </div>
            <div class="space-80"></div>
            <div class="border-_5px_white"></div>
            <div class="space-20"></div>
            <div class="welcome_text">
                Â© All rights reserved by the Yuanning-Li Lab
                <div class="alignright ">
                    Designed by Bo Feng
                </div>
            </div>
        </div>
    </div>
    <!---====footer-area area end====-->
    
    <!--:::::ALL JS FILES :::::::-->

    <script>
                    var Now = new Date(),
				   Timezone = Now.getTimezoneOffset()/60, //GMT æ¶é´ä¸æ¬å°æ¶é´ä»¥åéè®¡çæ¶é´å·®, æ¢æå°æ¶ï¼ nashvilleï¼UTC-5h; China: UTC +8
				   //The time-zone offset is the difference, in minutes, between UTC and local time. Note that this means that the offset is positive if the local timezone is behind UTC and negative if it is ahead. For example, if your time zone is UTC+10 (Australian Eastern Standard Time), -600 will be returned. Daylight savings time prevents this value from being a constant even for a given locale
				CurrentDay = Now.getDay(),
                OpeningTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 22, 00),
                ClosingTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 7, 00),
                Open = (Now.getTime() > OpeningTime.getTime() && Now.getTime() < ClosingTime.getTime());
				   //document.write('Today is ');
				   //document.write(Now);
				   //document.write('<br>');
				   //document.write('<br>');
				    //document.write(Timezone);
				    //document.write('<br>');
				    //document.write(CurrentDay);
				    //document.write('<br>');
				    //document.write(Now.getFullYear());
				    //document.write('<br>');
				    //document.write(Now.getMonth());
				    //document.write('<br>');
				    //document.write(Now.getDay());
				    //document.write('<br>');
				    //document.write(Now.getTime());
				    //document.write('<br>');
				    //document.write(OpeningTime.getTime());
				    //document.write('<br>');
				    // document.write(ClosingTime.getTime());
				    //document.write('<br>');
                    // days 0.sun 1.mon 2.tues 3.wed 4.thur 5.fri 6.sat 
                    // if (CurrentDay !== 6 && CurrentDay !== 0 && Open) {
					// if (Timezone == -8 && Open) {
				    //if (Timezone == -8 ) { // only display in China
                    //  document.write('<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">16. Kominek J.*, Doering D.T.*, Opulente D., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Jeremy D., Hulfachor A.B., Mcgee M., Karlen S., Kurtzman C.P., Rokas A., Hittinger C.T. 2019. Eukaryotic acquisition of a bacterial operon. <b><font face="verdana"><i>Cell</i></font></b> in press. (*co-first author)<div style="padding-left: 1em;"></div></p>');
                   //   }
				   // </script>
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


    <script src="assets/js/vendor/jquery.3.3.1.min.js"></script>
    <script src="assets/js/jquery.waypoints.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/js/vendor/bootstrap.min.js"></script>
    <script src="assets/js/vendor/owl.carousel.min.js"></script>
    <script src="assets/js/plugins/jquery-modal-video.min.js"></script>
    <script src="assets/js/plugins/stellarnav.js"></script>
    <script src="assets/js/plugins/jquery.nav.js"></script>
    <script src="assets/js/plugins/mesonry.mon.js"></script>
    <script src="assets/js/plugins/bars.js"></script>
    <script src="assets/js/main.js"></script>
</body>

</html>